.github/workflows/tests.yml

```diff
@@ -22,18 +24,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 6
+          node-version: 14
           cache: 'npm'
-      - run: npm install --no-audit && npm install --no-audit karma-cli karma-sauce-launcher
-      - name: Setup Sauce Labs
-        run: sh -e /etc/init.d/xvfb start
-        env:
-          DISPLAY: ':99.0'
-      - run: npm test
-      - run: ./node_modules/.bin/karma start karma.conf-sauce.js
+      - name: Install dependencies
+        run: |
+          npm install --no-audit
+          npm install --no-audit karma-cli karma-sauce-launcher
+      - name: Test
+        run: npm test
+      - name: Test in Sauce Labs
+        run: ./node_modules/.bin/karma start karma.conf-sauce.js
         env:
           SAUCE_USERNAME: ${{ secrets.SauceUsername }}
           SAUCE_ACCESS_KEY: ${{ secrets.SauceAccessKey }}
-          BUILD_NUMBER: ${{ GITHUB_RUN_NUMBER }}
-          BUILD_ID: ${{ GITHUB_RUN_ID }}
-          JOB_NUMBER: ${{ GITHUB_JOB }}
```

Comment:
We can workaround this by prependings vars to the command like

```
BUILD_NUMBER="$GITHUB_RUN_NUMBER" BUILD_ID="$GITHUB_RUN_ID" JOB_NUMBER="$GITHUB_JOB" ./node_modules/.bin/karma start karma.conf-sauce.js
```

This will keep sauce config env-neutral